src/services/routing.ts

```diff
@@ -138,3 +138,33 @@ export function buildServicesRoute(extraQueryParams?: UrlQueryMap): UrlQueryMap
     ...extraQueryParams,
   };
 }
+
+export function createAppUrl(path = '/explore', urlParams?: URLSearchParams): string {
+  return `/a/${pluginJson.id}${path}${urlParams ? `?${urlParams.toString()}` : ''}`;
+}
+
+export enum UrlParameterType {
+  DatasourceId = 'var-ds',
```

Comment:
Can be taken from variables.ts, right?

Reply:
```suggestion
  DatasourceId = `var-${VAR_NAME}`,
```

Not sure if this is possible with TS.

Reply:
Yea, it's not, but I switched to using a `const`. Good recommendation.